crypto/base64/base64.c

```diff
@@ -196,6 +196,8 @@ void EVP_EncodeUpdate(EVP_ENCODE_CTX *ctx, uint8_t *out, int *out_len,
     total = 0;
```

Comment:
We should consider returning a 0 here.  Looking at the OpenSSL implementation, they do return 0 if there is to much output data.